packages/compass-generative-ai/src/components/ai-optin-modal.tsx

```diff
@@ -58,29 +51,70 @@ const bannerStyles = css({
   marginTop: spacing[400],
   textAlign: 'left',
 });
-const getButtonText = (isOptInInProgress: boolean) => {
-  return (
-    <>
-      &nbsp;Use Natural Language
-      {isOptInInProgress && (
-        <>
-          &nbsp;
-          <SpinLoader darkMode={true}></SpinLoader>
-        </>
-      )}
-    </>
-  );
+
+// TODO: The LG MarketingModal does not provide a way to disable the button
+// so this is a temporary workaround to make the button look disabled.
+const focusSelector = `&:focus-visible, &[data-focus="true"]`;
+const hoverSelector = `&:hover, &[data-hover="true"]`;
+const activeSelector = `&:active, &[data-active="true"]`;
+const focusBoxShadow = (color: string) => `
+    0 0 0 2px ${color}, 
+    0 0 0 4px ${palette.blue.light1};
+`;
+const disabledButtonStyles: Record<Theme, string> = {
```

Comment:
Nit: we prefer object styles over templare literal ones, this is all throwaway temporary code, so you can keep it, just wanted to mention for the future